lib/sinon/spy.js

```diff
@@ -195,7 +196,7 @@ var spyApi = {
             if (thisCall.calledWithNew()) {
                 // Call through with `new`
                 // eslint-disable-next-line new-parens
```

Comment:
This line could probably have been removed with your addition.

Reply:
ah, true. thanks. will fix.